emscripten.py

```diff
@@ -1981,32 +2070,40 @@ def wrapped():
   args += ['--allow-memory-growth'] if shared.Settings.ALLOW_MEMORY_GROWTH else []
   args += ['-l', libc_rt_lib]
   args += ['-l', compiler_rt_lib]
+  args += ['-o', output]
+  return args
+
+
+def create_lld_args(temp_o, output):
+  def wasm_rt_fail(archive_file):
```

Comment:
Might as well combine this with the one in `create_s2wasm_args` and promote them to top-level functions